analytics_data_api/v0/models.py

```diff
@@ -336,7 +336,7 @@ def get_users_in_course(
             search.query.must.append(Q('multi_match', query=text_search, fields=['name', 'username', 'email']))
 
         # construct the sort hierarchy
-        search = search.sort(*[
+        search_request = search.sort(*[
```

Comment:
FYI, renaming this fixes a pylint `redefined-variable-type` error.  `search.sort` returns `elasticsearch_dsl.search.Request` instead of `elasticsearch_dsl.search.Search`.